templates/swagger/v1_json.tmpl

```diff
@@ -14926,6 +14937,12 @@
       "description": "CreateAccessTokenOption options when create access token",
       "type": "object",
       "properties": {
+        "Scope": {
```

Comment:
A capital letter?
Shouldn't this be `scope`?
And plural (`scopes`)?

Reply:
The field in the db is Scope which is where I've taken this from.